components/support/guid/src/lib.rs

```diff
@@ -315,6 +346,14 @@ impl fmt::Display for Guid {
     }
 }
 
+impl std::default::Default for Guid {
+    /// Create a default guid by calling `Guid::new("")`
+    #[inline]
+    fn default() -> Self {
+        Guid::new("")
```

Comment:
Nit: `Guid::empty()` (and in doc comment).

Reply:
Done!